twitterscraper/main.py

```diff
@@ -82,13 +82,10 @@ def main():
             logging.error("Output file already exists! Aborting.")
             exit(-1)
         
-        if args.lang:
-            args.query = "{}&l={}".format(args.query, args.lang)
-        
         if args.all:
             args.begindate = dt.date(2006,3,1)
 
-        tweets = query_tweets(args.query, args.limit, args.begindate, args.enddate, args.poolsize)
+        tweets = query_tweets(args.query, args.limit,args.lang, args.begindate, args.enddate, args.poolsize)
```

Comment:
I highly recommend to use keyword arguments here, this breaks easy and is nonexplicit (see also zen of python)

Reply:
Would it become something like ```query_tweets(args.query, ***vars(args))``` ?

Reply:
See commit https://github.com/taspinar/twitterscraper/pull/73/commits/c96b7423e9858feac729f8fa9bbab442fdca77df